functions.php

```diff
@@ -23,8 +23,128 @@ function tna_child_styles()
     wp_register_style('tna-parent-styles', get_template_directory_uri() . '/css/base-sass.css.min', array(),
         EDD_VERSION, 'all');
     wp_register_style('tna-child-styles', get_stylesheet_directory_uri() . '/style.css', array(), '0.1', 'all');
+	wp_register_style( 'tna-child-styles-cabinet-papers-100', get_stylesheet_directory_uri() . '/css/cabinet-papers-100.css', array(), '0.1', 'all' );
+	wp_register_style('tna-child-styles-cabinet-documents-child', get_stylesheet_directory_uri() . '/css/documents.css', array(), '0.1', 'all');
+	wp_register_style('font-awesome', 'https://maxcdn.bootstrapcdn.com/font-awesome/4.6.3/css/font-awesome.min.css', array(), '0.1', 'all');
+	wp_register_style( 'bx-slider-plugin-css', get_stylesheet_directory_uri() . '/js/lib/jquery.bxslider/jquery.bxslider.css', array(), '0.1', 'all' );
```

Comment:
This will be added to the feature 'removing bxslider', which will be added to the next release.